composer.json

```diff
@@ -13,9 +13,8 @@
     "require": {
         "php": "^7.1.3",
         "fideloper/proxy": "^4.0",
-        "laravel/framework": "5.7.*",
+        "laravel/framework": "5.8.*",
         "laravel/tinker": "^1.0",
-        "nwidart/laravel-modules": "^4.0",
```

Comment:
This needs to be here for auto-discovery to work. Otherwise, you have to manually load the service provider and facade for Laravel Modules in the app config.

`"nwidart/laravel-modules": "^5.0",`

Reply:
Thanks, I guess I removed it unintentionally.

Reply:
I have reverted to 4.0, 5.0 seems to be incompatible with other module or am I missing anything?